### DISCUSSION TITLE
Delete domain security policies

### DISCUSSION THREAD

**Discussion by BDenysovets** [bn-digital/home#22] [Environment] [answered]
### Domain security policy 

After working with secured local - **https:**//localhost:3000, in the new projects with **http:**//localhost:3000 you may have no error in the IDE console, but have some issues in the browser you are working with, which looks like `Your connection is not secure`, etc.

**Comment by vladyslavvolkov** [comment, marked as answer]
**How to resolve:**

- Go to [chrome://net-internals/#hsts](chrome://net-internals/#hsts)
- Scroll to `Delete domain security policies` section
- Input your current localhost into `Domain:` textfield & press delete
- Now your website with `HTTP://` work as expected

If it still not working:
- Go to `chrome://settings/clearBrowserData`
- Tick the box _Cached images_ and files and press click the button _Clear data_
